eth/_utils/module_loading.py

```diff
@@ -26,7 +26,7 @@ def import_string(dotted_path: str) -> ModuleType:
     try:
         return getattr(module, class_name)
     except AttributeError:
-        msg = 'Module "%s" does not define a "%s" attribute/class' % (
+        msg = 'Module "{}" does not define a "{}" attribute/class'.format(
```

Comment:
I wonder why it didn't migrate this (and some others) to an f-string?

Reply:
https://github.com/asottile/pyupgrade#f-strings

> note: pyupgrade is intentionally timid and will not create an f-string if it would make the expression longer or if the substitution parameters are anything but simple names or dotted names (as this can decrease readability).

I'm guessing this means they don't attempt multi-line fixups either?

Reply:
still an improvement.  gonna hold on merge to see about getting the other f-strings done either myself or via a bounty.